apps/backend/src/main/java/no/nav/data/team/graph/GraphService.java

```diff
@@ -63,6 +80,22 @@ public void deleteTeam(Team team) {
         }
     }
 
+    private void cleanupPrevCluster(Team team, String teamVertexId) {
+        var clusterIds = team.getClusterIds();
+
+        clusterIds.forEach(clusterId -> {
+            var clusterVertexId = VertexLabel.Cluster.id(clusterId);
+
+            var existingClusterVertex = client.getVerticesForEdgeOut(teamVertexId, EdgeLabel.partOfCluster);
+            if (!existingClusterVertex.isEmpty()
+                    && (existingClusterVertex.size() > 1 || !existingClusterVertex.get(0).getId().equals(clusterVertexId))
```

Comment:
Dette blir krøll, du må sjekke alle clusterne samtidig, ikke en og en, du ender opp med å slette alle andre links enn den siste

Reply:
(tanken for koden på produkt område er at det er maks 1, så om vi finner flere enn 1 link (skal ikke gå men greit å sjekke) eller om linken ikke er den nye så sletter vi